### PR TITLE
issue/968/Fix createsuperuser

### PR DESCRIPTION
**Pull request by tdelatorre**
Fix createsuperuser and fix empty email

### CODE REVIEW

File: taiga/users/models.py [hunk 1]
```diff
@@ -91,6 +91,39 @@ def get_user_file_path(instance, filename):
...
+        Creates and saves a superuser with the given username, email, password.
+        """
+        user = self.create_user(
+            username=username,
+            email=email,
+            password=password
+        )
+        user.is_superuser = True
+        user.is_admin = True
+        user.save(using=self._db)
```
Comment (yamila-moreno) [inline taiga/users/models.py:123]: why the param "using=self._db" ?

File: taiga/users/models.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 from django.apps import apps
 from django.apps.config import MODELS_MODULE_NAME
 from django.conf import settings
-from django.contrib.auth.models import UserManager, AbstractBaseUser
```
Comment (yamila-moreno) [inline taiga/users/models.py:28]: UserManager is no longer needed

File: taiga/users/models.py (hunk 1 above)
Comment (tdelatorre) [reply taiga/users/models.py:123]: If someone has another database defined in their config.py this uses the default. Is the same as using=None but I don't think it's necessary. I remove it.

File: taiga/users/models.py (hunk 2 above)
Comment (tdelatorre) [reply taiga/users/models.py:28]: Thanks! I remove it.

**Pull request merged**